ui/raidboss/data/03-hw/pvp/shatter.js

```diff
@@ -56,4 +56,64 @@
       },
     },
   ],
+  timelineReplace: [
+    {
+      'locale': 'de',
+      'missingTranslations': true,
+      'replaceSync': {
+        'The icebound tomelith A1 activates and begins to emit heat': 'Vereister Echolith A1 activates and begins to emit heat',
```

Comment:
Are these actually correct translations? They've always looked suspicious to me.

Reply:
i will need to run PvP for it to double check (all my logfiles dont contain the line when they spawn)